tests/auto_run_tests.pl

```diff
@@ -227,9 +222,12 @@ sub print_help {
 my $list_tests = 0;
 my $cmake = 0;
 my $cmake_build_dir = "$cmake_tests/build";
+my $cmake_build_cfg = windows ? 'Debug' : undef;
 my $ctest = 'ctest';
 my $ctest_args = '';
-my $python = 'python3';
+# Python on Windows is called python from what I can see. On other platforms
+# that are legacy-mindful python is Python 2, but we can rely on python3.
+my $python = windows ? 'python' : 'python3';
```

Comment:
`python3` doesn't appear to be part of Python itself according to https://docs.python.org/3/using/cmdline.html
It may be a change the distro packagers make.  That doesn't really impact what we need to do, but I think this is the other way around from what the comment says.  Some platforms that encourage every single program on the computer to share a common directory location need to resolve naming conflicts.  Other platforms let different things exist in different directories and users can decide what to run explicitly or with `PATH`.

Reply:
I think that page is more focused on the interpreter arguments. `python3` is definitely part of Python. In fact looking at the builds for Linux I did the other day, there is no `python` there, just `python3` which symlinks to `python3.9`.

Reply:
Looks like `make` builds an interpreter called `python`, but the `make install` process renames it.  It's unfortunate that the page of their documentation that seemed to be the most relevant doesn't cover this.